src/client/core/utils/content-editable.js

```diff
@@ -1,40 +1,40 @@
-import $ from '../deps/jquery';
 import * as domUtils from './dom';
+import * as serviceUtils from './service';
 
 
 //nodes utils
 function getOwnFirstVisibleTextNode (el) {
-    var childrenArray = $.makeArray(el.childNodes),
-        child         = null;
+    var children = el.childNodes;
+    var child    = null;
 
-    if (!childrenArray.length && el.nodeType === 3 && !isInvisibleTextNode(el))
+    if (!children.length && el.nodeType === 3 && !isInvisibleTextNode(el))
         return el;
 
-    $.each(childrenArray, function (index, value) {
-        if (value.nodeType === 3 && !isInvisibleTextNode(value)) {
-            child = value;
-            return false;
+    for (var i = 0; i < children.length; i++) {
```

Comment:
It's good to cache NodeList length, since it's live property and causes unnecessary recalculations. Btw, maybe we should implement `forEachChildNode` in utils?